### ISSUE TITLE
maskiou head的l2_loss中的gt_maskiou是什么？

### ISSUE THREAD

**Issue by Elsanna** [zjhuang22/maskscoring_rcnn#62]
在代码中，maskiou_loss = l2_loss(pred_maskiou[positive_inds, labels_pos], gt_maskiou)
pred_maskiou是maskiou_head输出的对于iou的预测值，那么gt_maskiou是什么呢？
gt为什么会有iou...很疑惑，求大神解答。

**Comment by zjhuang22** [comment]
您好，预测的gtiou就是  predicted mask和gt mask的iou，用来监督pred_maskiou的学习

**Comment by xs-alt** [comment]
您好，为什么ground truth mask area 是这样计算出来的
mask_targets_full_area = mask_targets.sum(dim=[1,2]) / mask_ratios

**Comment by chongruo** [comment]
@bringBackm  因为原来的 groundtruth 只是 proposal 里的 mask (object 的部分 mask ), 而不是 object 完整的 mask。 在 mask score 的 target generation 中，需要用 object 完整的 mask 来计算最后的 mask IOU。所以这里用 mask_ratio 修正了一下。
